database/README.md

```diff
@@ -4,4 +4,4 @@
 Running `make dev` in the docker environment will create and persist the database.
 
 ## Database Diagram
-![](database_relationships.Sept2023.png)
+![](database_relationships.July2024.png)
```

Comment:
I updated the database relationship diagram so that it's in-sync with the current state of things.

Reply:
thanks for doing that the updated database diagram, I think that's super helpful for contributors.